sdk/python/feast/type_map.py

```diff
@@ -155,6 +155,7 @@ def python_type_to_feast_value_type(
         "uint16": ValueType.INT32,
         "uint8": ValueType.INT32,
         "int8": ValueType.INT32,
+        "bool_": ValueType.BOOL,
```

Comment:
what is this type `bool_`?

Reply:
It's a numpy bool type, but harder to identify since we use `type(value).__name__`, _e.g._

```Python
>>> import pandas as pd
>>> data = pd.DataFrame({"a": [True, False]})
>>> type(data["a"].iloc[0])
<class 'numpy.bool_'>
>>> type(data["a"].iloc[0]).__name__
'bool_'
```

We need this mapping for boolean features in pandas transformations.

Reply:
gotcha. add a simple doc would be better.